packages/core/cli.js

```diff
@@ -63,6 +63,16 @@ const command = getCommand({
   options: {},
   noAliases: false
 });
+
+//getCommand() will return null if a command not recognized by truffle is used.
+if (command === null) {
+  let err = new Error(
```

Comment:
If we are not going to throw this `Error`, there is no sense in creating it. Just log and `exit`.

Reply:
okay. will be making the change.